index.html

```diff
@@ -0,0 +1,499 @@
+                height="50"
+                alt="riezh-robert"
+              />
+              <div class="trusted-customer__username">
+                <p class="trusted-customer__name">Viezh Robert</p>
+                <p class="trusted-customer__born">Poland</p>
+              </div>
+              <div class="trusted-customer__rating">
+                <p class="trusted-customer__text">4.5</p>
+                <img src="./img/star.svg" alt="star" width="16" height="16" />
+              </div>
+            </div>
+            <blockquote class="trusted-customer--text">
+              “Wow... I am very happy to use this VPN, it turned out to be more
+              than my expectations and so far there have been no problems.
+              LaslesVPN always the best”.
+            </blockquote>
+          </div>
+          <div class="trusted-customer__item">
+            <div class="trusted-customer--datauser">
+              <img
+                src="./img/yessica-christy.svg"
+                width="50"
+                height="50"
+                alt="yessica-christy"
+              />
+              <div class="trusted-customer__username">
+                <p class="trusted-customer__name">Yessica Christy</p>
+                <p class="trusted-customer__born">Shanxi, China</p>
+              </div>
+              <div class="trusted-customer__rating">
+                <p class="trusted-customer__text">4.5</p>
+                <img src="./img/star.svg" width="16" height="16" alt="star" />
+              </div>
+            </div>
+            <blockquote class="trusted-customer__text">
+              “I like it because I like to travel far and still can connect with
+              high speed.”.
+            </blockquote>
+          </div>
+          <div class="trusted-customer__item">
+            <div class="trusted-customer--datauser">
+              <img
+                src="./img/kim-young-jou.svg"
+                width="50"
+                height="50"
+                alt="kim-young-jou"
+              />
+              <div class="trusted-customer__username">
+                <p class="trusted-customer__name">Kim Young Jou</p>
+                <p class="trusted-customer__born">Seoul, South Korea</p>
+              </div>
+              <div class="trusted-customer__rating">
+                <p class="trusted-customer__text">4.5</p>
+                <img src="./img/star.svg" width="16" height="16" alt="star" />
+              </div>
+            </div>
+            <blockquote class="trusted-customer__text">
+              “This is very unusual for my business that currently requires a
+              virtual private network that has high security.”.
+            </blockquote>
+          </div>
+        </div>
+        <div class="trusted-customer__control">
+          <button class="button button__left" type="button">left</button>
+          <button class="button button__right" type="button">right</button>
```

Comment:
Here should be `trusted-customer__button button` rather than plain `button`.
`__left` / `__right` look like modifiers, you'd better replace `__` with `--` here, so that the whole like looks like:
```
trusted-customer__button button button--left
```

Reply:
+